spring-boot-3-integrationtests/pom.xml

```
@@ -39,7 +39,7 @@
         <mysql-connector-java.version>8.0.33</mysql-connector-java.version>
         <spring-boot-3.version>3.1.5</spring-boot-3.version>
         <!-- Build / Plugin -->
-        <jacoco-maven.version>0.8.8</jacoco-maven.version>
+        <jacoco-maven.version>0.8.11</jacoco-maven.version>
```

Ah wait; this will then fail on Java 8 it seems; can't win.
```suggestion
        <jacoco-maven.version>0.8.8</jacoco-maven.version>
```

---

Indeed now the only failures are that Spring Boot Docker Compose issue. Thanks!